tests/Functional/ContentProjectRenderTest.php

```diff
@@ -18,6 +21,13 @@ class ContentProjectRenderTest extends BrowserTestBase {
    */
   protected $nodeStorage;
 
+  /**
+   * The organisation storage.
+   *
+   * @var \Drupal\oe_content_entity_organisation\Entity\OrganisationInterface
+   */
+  protected $organisationStorage;
```

Comment:
I see this called only once, do we need it as a class property ?

Reply:
I agree, let's use a local variable.

Reply:
Antonio asked to return it back.